sparrowpy/classes/RadiosityFast.py

```diff
@@ -1242,12 +1269,12 @@ def get_scattering_data_receiver_index(
         pos_i:np.ndarray, pos_j:np.ndarray,
         receivers:np.ndarray, wall_id_i:np.ndarray,
         ):
-    """Get scattering data depending on previous, current and next position.
+    """Get scattering receiver index based on current and next position.
```

Comment:
```suggestion
    """Get scattering receiver index based on current and next position.
    
    Note that all indexes on the same wall get an invalid entry, namly n_patches.
```
maybe add a note in the doc here?

Reply:
will ignore this suggestion in this line, since the function is no longer processing coincident pos_i and pos_j points